A5hleyRich/wp-queue
# Update dependencies and PHP version

This PR:
 - updates the dependencies (in particular Carbon v2) - resolves #12 
 - updates the required PHP version to 7.3 (as required by Carbon v2.0 and PHPUnit v9.0)
 - adds WPCS and PHPStan
 - prepares for submission to packagist.org as deliciousbrains/wp-queue 

All tests pass

There are some PHPCS and PHPStan fails that are not critical and will mostly need manual fixing.

## Testing

A code review should show there's no functional changes. You can:

1. Run `./vendor/bin/phpunit`
2. Run `./vendor/bin/phpstan analyze` (there will be errors, but nothing that affects functionality)
